File: C# Part 2/03.Methods/04.AppearanceCount.cs
using System;
using System.Linq;

namespace AppearanceCount
{
    class AppearanceCount
    {
        static void Main()
        {
            int sizeOfArray = Convert.ToInt32(Console.ReadLine());
            int[] input = Console.ReadLine().Split(' ').Select(x => Convert.ToInt32(x)).ToArray();
            int number = Convert.ToInt32(Console.ReadLine());
            Console.WriteLine(AppCount(number, input));
        }

        static int AppCount(int pattern, int[] input)
        {
           return input.Where(x => x== pattern).Count();
        }
    }
}

File: JavaScript Fundamentals/08. Functions/04_AppearanceCount.js
function solve(args) {
    var n = +args[0];
    var x = +args[2];
      
    args = args[1].split(' ');

    var count = 0;
    for (var i = 0; i < n; i += 1) {
        if (x === +args[i]) {
            count += 1;
        }
    }

    console.log(count);
}


solve(['8\n28 6 21 6 -7856 73 73 -56\n73']);

